src/main/java/com/example/JavaProject/entity/RecipeIngredient.java

```diff
@@ -4,24 +4,20 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
-import jakarta.validation.constraints.Min;
-import jakarta.validation.constraints.NotNull;
 
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
 @Entity
-@Table(name = "recipe_ingredient")
+@Table(name = "recipes_ingredients")
```

Comment:
I prefer singular naming convention for database tables

Reply:
Other than that everything is fine

Reply:
Okey then, I changed them back to singular

Right now Is everything okey?

Reply:
Yes, you can go ahead and merge it to master